modules/pico_tftp.c

```diff
@@ -13,8 +13,11 @@
 #include <pico_tftp.h>
 #include <pico_strings.h>
 
-#define tftp_dbg(...) do {} while (0)
-//#define tftp_dbg dbg
+#ifdef DEBUG_TFTP
+#define tftp_dbg dbg
+#else
+#define tftp_dbg(...) do {} while(0) /* so_mcast_dbg in pico_socket.c */
```

Comment:
Indeed, copy-paste error and some laziness I guess.
Should be fixed in 447fdd92ce406233178548df499415a2c566df2e